es/home/src/main/resources/rdf/i18n/es/display/firsttime/aboutQrCodes.n3

```diff
@@ -0,0 +1,38 @@
+# $This file is distributed under the terms of the license in LICENSE$
+
+@prefix about: <http://vitro.mannlib.cornell.edu/ns/default/about#> .
+
+about:about_qr_codes_dg
+    a   <java:edu.cornell.mannlib.vitro.webapp.utils.dataGetter.FixedHTMLDataGetter> ;
+    <http://vitro.mannlib.cornell.edu/ontologies/display/1.1#htmlValue>
+        """
+        <h2>Acerca de los c&oacute;digos QR</h2>
+
+		<h3>Qu&eacute; es QR?</h3>
+
+		<p>QR, siglas de <i>Quick Response</i>, es un est&aacute;ndar para los c&oacute;digos de barras de dos dimensiones que pueden ser le&iacute;dos por los dispositivos m&oacute;viles equipados con c&aacute;maras (como un tel&eacute;fono inteligente).</p>
+
+		<h3>Qu&eacute; tipo de dato es un c&oacute;digo QR?</h3>
+
+		<p>Los c&oacute;digos QR pueden codificar diferentes tipos de datos. El c&oacute;digo QR de la p&aacute;gina del perfil de una persona en VIVO codifica la informaci&oacute;n acerca de esa persona en formato <i>vCard</>. Se trata de una especie de tarjeta de presentaci&oacute;n digital, que contiene informaci&oacute;n sobre el nombre de la persona, direcci&oacute;n, n&uacute;mero de tel&eacute;fono, direcci&oacute;n de correo electr&oacute;nico y mucho m&aacute;s.</p>
```

Comment:
closing i tag is missing </i>, therefore the rest of the text is presented as italic in web browser

Reply:
fixed